inst/rmarkdown/lua/latex-div.lua

```diff
@@ -4,6 +4,11 @@
      License: Public domain
 --]]
 
+if (not PANDOC_VERSION) or (PANDOC_VERSION < "2.1") then
+    print("latex-div.lua requires Pandoc > 2.1")
```

Comment:
After some thoughts I would use `io.stderr:write` for these. We may also choose a layout that make it unique and recognizable as a warning / log string

Something like 
```
[WARNING](from latex-div.lua) requires Pandoc > 2.1. Lua filter has been ignored and some features disabled
```
What do you think ? 
```suggestion
    io.stderr:write("[WARNING] (latex-div.lua) requires at least Pandoc 2.1")
```

Reply:
I like it

Reply:
To add to that, I am wondering if it would be useful to add more info on the message. like 

````
io.stderr:write("[WARNING] (latex-div.lua) requires at least Pandoc 2.1. Lua Filter has been ignored and some features hence disabled."
```

Too much ?

Reply:
I agree previous warning message is not kind enough because it does not explain what happens.
I think we can simply say "skipping" the filer, which implies disabling some features.

```
io.stderr:write("[WARNING] Skipping (latex-div.lua) which requires at least Pandoc 2.1."
```

Reply:
Like the requirement first which is the cause, then the effect it has. 
Maybe 
```
io.stderr:write("[WARNING] (latex-div.lua) requires at least Pandoc 2.1. Lua Filter skipped")
```

Reply:
got it, and done it by 2105f6d.